### ISSUE TITLE
Naming scheme changed in daily reports for 03-23-2020.

### ISSUE THREAD

**Issue by Spandan-Madan** [CSSEGISandData/COVID-19#1385]
It seems column name has been changed from Country/Region to Country_Region on the daily report of 03-23-2020.

Is this part of planned change? Or error?

Thought I should flag it!

**Comment by JcMinarro** [comment]
The columns positions have changed as well, breaking parsers

**Comment by jscudder** [comment]
Also, time series now leaves out US States? Sad. Completely breaks my racing bar chart http://covid19viz.azurewebsites.net/covid19/covid19barchartrace.html

**Comment by AaronWard** [comment]
@CSSEGISandData If you are going to change column names, make sure the columns are the same across all CSVs and not just one. And please notify people in advance when you are doing it.

People are using this datasources for their own dashboards and these unannounced changes will break them

**Comment by kenlog** [comment]
I believe I should develop a new parser. You see #1250 :confused:

**Comment by JcMinarro** [comment]
@kenlog It doesn't say anything about "Daily report scheme", only for "Time Series".

**Comment by kenlog** [comment]
I agree @JcMinarro this change is probably not mentioned but the file itself confirms it.

**Comment by cipriancraciun** [comment]
I have created a derived dataset, available in JSON and TSV (in a more SQL friendly format), where I tried to solve some of these issues (like normalizing country names, providing dates in ISO format, etc.), plus I tried to compute simple metrics, and augment it with additional information (like continent, sub-continent, country totals, day index since N number of confirmed cases, etc.):
* https://github.com/cipriancraciun/covid19-datasets -- my derived dataset repository;
* https://github.com/CSSEGISandData/COVID-19/issues/1281 -- the issue on this repository where I describe it;

**Comment by kenlog** [comment]
I would point out that [web-data](https://github.com/CSSEGISandData/COVID-19/tree/web-data/data) branch would be an alternative. In my case, adapting is easier, I hope for others too.
